internal/driver/commands.go

```diff
@@ -131,6 +131,12 @@ var pprofVariables = variables{
 	// Filename for file-based output formats, stdout by default.
 	"output": &variable{stringKind, "", "", helpText("Output filename for file-based outputs")},
 
+	"comment": &variable{stringKind, "", "", helpText(
```

Comment:
Not sure this should be a variable.

If somebody were to set comment="my comment" on the pprof shell it would have no effect, right?

Seems to me more like -buildid, in that it only affects the profile during fetch. Consider parsing it explicitly on parseFlags instead.

Reply:
Made it a flag similar to what `-buildid` is.